samples/xamarin-forms/Cinephile/Cinephile/Views/AboutView.xaml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui:ContentPageBase xmlns="http://xamarin.com/schemas/2014/forms" 
+                    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" 
+                    x:Class="Cinephile.Views.AboutView"
+                    xmlns:ui="clr-namespace:Cinephile.Views"
+                    xmlns:vms="clr-namespace:Cinephile.ViewModels"
```

Comment:
Warning here! We have now a separated project for ViewModels and you *MUST* specify the assembly `assembly=Cinephile.ViewModels`. It would look like below:

```
xmlns:vms="clr-namespace:Cinephile.ViewModels;assembly=Cinephile.ViewModels"
```